include/constants/snd_def.h

```diff
@@ -0,0 +1,17 @@
+#ifndef POKEHEARTGOLD_SND_DEF_H
+#define POKEHEARTGOLD_SND_DEF_H
+
+#define BGM_VOL_MAX                    127                            
+#define BGM_VOL_HALF                    64                            
+#define BGM_VOL_MIN                      0                                
+#define BGM_VOL_ZUKAN      BGM_VOL_MAX / 3                
```

Comment:
Unless the names come from somewhere baked into the ROM, Japanese names should be translated to English. For example, this would be `BGM_VOL_POKEDEX`.